PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchRemoteRunState.kt

```diff
@@ -71,7 +72,9 @@ open class SparkBatchRemoteRunState(val serverlessSparkSubmitModel: SparkSubmitM
                         err.printStackTrace(PrintWriter(errWriter))
 
                         val errMessage = Optional.ofNullable(err.message)
-                                .orElse(err.toString()) + "\n stack trace: " + errWriter.buffer.toString()
+                                .orElse(err.toString())
+
+                        log().error(errMessage + "\n stack trace: "+ errWriter.buffer.toString())
```

Comment:
I suppose `log().warn()` is enough. The `error()` log will cause Intellij report an issue to the user.